pkg/controller/nodenetworkconfigurationpolicy/policyconditions/conditions.go

```diff
@@ -24,7 +24,7 @@ func setPolicyProgressing(conditions *nmstatev1alpha1.ConditionList, message str
 		nmstatev1alpha1.NodeNetworkConfigurationPolicyConditionDegraded,
 		corev1.ConditionUnknown,
 		nmstatev1alpha1.NodeNetworkConfigurationPolicyConditionConfigurationProgressing,
-		"",
+		message,
```

Comment:
I get why you would look for this message when "unknown", but I'd rather drop it to keep message only in a single condition. Plus it is not related to this PR. Once we have solid basics, we can optimize messages and stuff like that.